O365/utils/utils.py

```diff
@@ -636,6 +636,9 @@ class Query:
         'flag': 'flag/flagStatus',
         'body': 'body/content'
     }
+    _select_mapping = {
+        "meetingMessageType": "microsoft.graph.eventMessage/meetingMessageType",
```

Comment:
This is wrong as this info which uses a keyword referencing ms graph it's owned by the protocol.

This keyword is currently stored in the protocol.

Check this [line](https://github.com/O365/python-o365/blob/master/O365/utils/query.py#L734) for example. 

So instead it should do something like this:

```python
attribute = f"{self.protocol.keyword_data_store['event_message_type']}/meetingMessageType"
```

Reply:
OK, this depends on new query class. I think I should wait until you have that in place, since you will be changing other calls to it. I can redo the PR when it is ready.

Reply:
Or I can make it inline with the old version of the class, which I can see handles it in 'expand'.

Reply:
Whatever you prefer. Anyway you can add this also to the new Query class

Reply:
> Whatever you prefer. Anyway you can add this also to the new Query class

I've left it in Utils, let me know if it should be move to Query. Where you linked to in Query was for an Expand, whilst this is a select.

Reply:
This should be added as well for the new query.

Reply:
I'm not sure how you mean to include it there, do you mean in here:

```python
    def select(self, *attributes: str) -> SelectFilter:
        """
        Returns a 'select' query param
        This is useful to return a limited set of attributes from a resource or return attributes that are not
         returned by default by the resource.

        :param attributes: a tuple of attribute names to select
        :return: a SelectFilter instance that can render the OData select operation
        """
        select = SelectFilter()
        for attribute in attributes:
            select.append(self.protocol.convert_case(attribute))
        return select
```

Not sure how to test it, but I can add the same logic. Shall I open a new PR?

Reply:
yeah it should be there. But I think you can also expand on this attribute?
Maybe this should work both for select and expand.

Also note that I think it's better if the check is done ignoring the casing of the provided attribute like:

`if attribute.lower() == "meetingmessagetype"`

Reply:
> But I think you can also expand on this attribute?

No - this error

`"Parsing OData Select and Expand failed: Property 'meetingMessageType' on type 'microsoft.graph.eventMessage' is not a navigation property or complex property. Only navigation properties can be expanded."`

Reply:
> > But I think you can also expand on this attribute?
> 
> No - this error
> 
> `"Parsing OData Select and Expand failed: Property 'meetingMessageType' on type 'microsoft.graph.eventMessage' is not a navigation property or complex property. Only navigation properties can be expanded."`

oh. Ok!!